snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: kata-containers
-version: "1.9.0-alpha0"
+version: "master"
```

Comment:
Is the plan to change it to the actual version before we release a snap, or always leave it on `master`? I do wonder if it would make sense to rename this `snapcraft.yaml.in` and generate this field.

Reply:
testing on `master` sounds good.
If it makes more sense to leave the actual snap release in this file, maybe the the CI can substitute for `master` at run time....
or, maybe we encode the snap version into our versions.yaml, and have something pull/inject it into this file at snap build time.

Reply:
> I do wonder if it would make sense to rename this snapcraft.yaml.in and generate this field.

This was the first approach used when we started with this, unfortunately launchpad (that is our build system) is not able to pre-process this file to generate the final `snapcraft.yaml` and it cannot be configured to run custom commands (like `make snap`), launchpad just pulls the code and runs `snapcraft -d` to generate the snap.

@jodh-intel  @grahamwhaley  @ganeshmaharaj 

In order to have a better control over the snaps that we release, detect errors earlier and allow users try the latest of Kata Containers (master branch). I have created two different project in launchpad: [snap-master](https://launchpad.net/~kata-containers-dev/+snap/snap-master) and [snap-stable](https://launchpad.net/~kata-containers-dev/+snap/snap-stable). The first project will generate a snap based on the `master` branch (we need this change) and will release it on the `edge` channel, while the second one will generate the snap based on the `snap-next` branch and will release on the stable channel.